doc/source/notebooks/theory/Sanity_check.ipynb

```diff
@@ -67,61 +72,27 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 3,
+   "execution_count": 8,
```

Comment:
Would be good to re-run the notebook with "Restart & Run All" before merging, so that the cell numbers are nicely consecutive (otherwise it can appear confusing to people reading through the notebook!).

Reply:
...okay :)

Reply:
done